### PR TITLE
Kubernetes operator configuration doc

### PR DESCRIPTION
**Pull request by rakillen**

### CODE REVIEW

File: site/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+apiVersion: weblogic.oracle/v6
+kind: Domain
+metadata:
+    name: DemoDomain
+spec:
+    domainHome: /u01/mydomain
+    imagePullSecrets:
+    -   name: '--FIX ME--'
+    webLogicCredentialsSecret: '--FIX ME--'
+    clusters:
```
Comment (ddsharpe) [inline site/kubernetes.md]: since "image" is a required attribute under spec, why doesn't the tool generate a placeholder for this?

Comment (rakillen) [reply site/kubernetes.md]: Added in latest commit, code changes covered in PR #521

File: site/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+## Configuring Oracle WebLogic Server Kubernetes Operator
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
+
+The Extract Domain Resource Tool is available with WDT releases 1.7.0 and later.
+
+More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://github.com/oracle/weblogic-kubernetes-operator).
```
Comment (rosemarymarano) [inline site/kubernetes.md]: More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://github.com/oracle/weblogic-kubernetes-operator). Do you prefer to send your readers to the operator repository versus the docs, https://oracle.github.io/weblogic-kubernetes-operator?

File: site/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+Here is an example command line for the Extract Domain Resource Tool:
+```
+<wls-deploy-home>/bin/extractDomainResource.sh -oracle_home /tmp/oracle	-domain_home /u01/mydomain -model_file /tmp/mymodel.yaml -domain_resource_file /tmp/operator/domain-resource.yaml
+```
+
+For the simplest case, the Extract Domain Resource Tool will create a sparse domain file. This is what is generated when there is not `kubernetes` section in the model, or that section is empty.
```
Comment (rosemarymarano) [inline site/kubernetes.md]: there is not `kubernetes` section -> there is not a `kubernetes` section

File: site/kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    -   name: '--FIX ME--'
+    webLogicCredentialsSecret: '--FIX ME--'
+    clusters:
+    -   clusterName: mycluster
+        replicas: 2
+    -   clusterName: mycluster3
+        replicas: 4
+```
+
+For this example, the user is expected to fill in the image and secrets information identified by `--FIX ME--` in the domain resource output. The value for `domainHome` was set from the command line. The `kind` and `name` were set to the domain name derived from the topology section of the model, or the default `base_domain`. The cluster entries are pulled from the topology section of the model, and their replica counts were derived from the number of servers for each cluster.
```
Comment (rosemarymarano) [inline site/kubernetes.md]: For this example, -> In this example,

File: site/kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
+
+The Extract Domain Resource Tool is available with WDT releases 1.7.0 and later.
+
+More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://github.com/oracle/weblogic-kubernetes-operator).
+
+Here is an example command line for the Extract Domain Resource Tool:
+```
+<wls-deploy-home>/bin/extractDomainResource.sh -oracle_home /tmp/oracle	-domain_home /u01/mydomain -model_file /tmp/mymodel.yaml -domain_resource_file /tmp/operator/domain-resource.yaml
```
Comment (mriccell) [inline site/kubernetes.md]: I think we should include the properties file as well in this example

File: site/kubernetes.md (hunk 4 above)
Comment (mriccell) [inline site/kubernetes.md]: I think that we should guide users to replace --FIX ME-- as ' @@PROP:param@@' in a properties file.

File: site/kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+## Configuring Oracle WebLogic Server Kubernetes Operator
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
```
Comment (mriccell) [inline site/kubernetes.md]: We should emphasize that this specially interesting when making configuration changes that are also need to be reflected in the domain Custom Resource.  Example when adding a cluster to the domain just make the changes to the WDT model section that represents the WLS configuration and you can generate a Customer Resource yaml which you can apply to kubernetes.

File: site/kubernetes.md (hunk 4 above)
Comment (rakillen) [reply site/kubernetes.md]: Agreed, I'll add it to the sample model with some details

File: site/kubernetes.md (hunk 6 above)
Comment (rakillen) [reply site/kubernetes.md]: I'll add this, borrowing heavily from this comment!

File: site/kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+This is especially useful when making configuration changes to the domain that also need to be reflected in the domain resource file. For example, adding a cluster to the domain only requires that it be added to the `topology` section of the WDT model, then a new domain resource file can be generated to apply to Kubernetes.
+
+More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://oracle.github.io/weblogic-kubernetes-operator).
+
+NOTE: The Extract Domain Resource Tool is available with WDT releases 1.7.0 and later.
+
+Here is an example command line for the Extract Domain Resource Tool:
+```
+<wls-deploy-home>/bin/extractDomainResource.sh -oracle_home /tmp/oracle	-domain_home /u01/mydomain -model_file /tmp/mymodel.yaml -variable_file /tmp/my.properties -domain_resource_file /tmp/operator/domain-resource.yaml
```
Comment (mriccell) [inline site/kubernetes.md]: Formatting in this line there are several blank spaces between /tmp/oracle and -domain_home

File: site/kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+    -   name: '--FIX ME--'
+    webLogicCredentialsSecret: '--FIX ME--'
+    clusters:
+    -   clusterName: mycluster
+        replicas: 2
+    -   clusterName: mycluster3
+        replicas: 4
+```
+
+In this example, the value for `domainHome` was set from the command line. The `kind` and `name` were set to the domain name derived from the topology section of the model, or the default `base_domain`. The cluster entries are pulled from the topology section of the model, and their replica counts were derived from the number of servers for each cluster.
```
Comment (mriccell) [inline site/kubernetes.md]: as an input parameter to the extractDomainResource.sh from the command line.

**Pull request merged**